### PR TITLE
Steps towards python3 

### PR DESCRIPTION
**Pull request by vodik**
Switch to proper relative imports so that the code will import, and then add tox for testing.

Major outstanding changes needed to support python3 seem to revolved around `values()` call, which under python3 are now views.

### CODE REVIEW

**Comment by vodik** [comment]
Tests pass on python 3 now, so that's something.

File: setup.py [hunk 1]
```diff
@@ -38,9 +38,8 @@
     platforms=['linux'],
     packages=['pysipp'],
     install_requires=['pluggy==0.3.1'],
-    extras_require={
-        'testing': ['pytest'],
-    },
+    setup_requires=['pytest-runner'],
```
Comment (tgoodlet) [inline setup.py]: I like it, but maybe we should adjust to [avoid installation on each `setup.py` call](http://pythonhosted.org/pytest-runner/#considerations)?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+language: python
```
Comment (tgoodlet) [inline .travis.yml]: I'm assuming you've already set up the build yourself?
If so we should get a [status link](https://docs.travis-ci.com/user/status-images/) to it in the README.

File: setup.py (hunk 1 above)
Comment (vodik) [inline setup.py]: I'll look into it

File: .travis.yml (hunk 2 above)
Comment (vodik) [inline .travis.yml]: I don't have enough permissions to join the project to travis. Meaning that I can't test this and I can't get the link (well, its craftable, but I don't want to add it without a test).

If you can register this project on travis, I'll do it for you.

Comment (tgoodlet) [inline .travis.yml]: Ok I'll get it set up.

File: tox.ini [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+# in multiple virtualenvs. This configuration file will run the
+# test suite on all supported python versions. To use it, "pip install tox"
+# and then run "tox" from this directory.
+
+[tox]
+envlist = py27, py34, py35, pypy
+
+[testenv]
+commands =
+	{envpython} setup.py install
```
Comment (tgoodlet) [inline tox.ini]: Pretty sure this is done automatically.
Not sure if you put it there explicitly for some reason?

Comment (vodik) [inline tox.ini]: Fixed.

**Pull request merged**